client/src/hooks/useSearchResources.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useSearch } from './useSearch'
+import { ResourcesPortalContext } from '../ResouresPortalContext'
```

Comment:
```suggestion
import { ResourcesPortalContext } from '../ResourcesPortalContext'
```
This will necessitate a file change as well.